### PR TITLE
Showing 120% score in exam report

### LINKED ISSUE #3058: Showing 120% score in exam report
### Observed behavior
After submitting exam, when coach user watching progress of each user in exam report. Coach user see 120% score in exam report. We have attached screenshot and database file,so you can easily re-generate this issue.

### Expected behavior
Score must be between 0-100%.

### Steps to reproduce
1. Copy attached database file in .kolibri folder.
2. login with username "pm" and password "sc".
3. Click on Coach.
4. Click on Class 4A.
5. Click on Exams.
6. See report of the Unit 2B-Final exam.
7. See learner Junaid Shaikh.

### Context
 * Kolibri version : Kolibri 0.4.9
 * Operating system : Ubuntu 14.04
 * Browser : Chrome 

### Database
[db.sqlite3.zip](https://github.com/learningequality/kolibri/files/1617728/db.sqlite3.zip)

### PR DESCRIPTION
**Pull request by mrpau-richard**
### Summary
I filter those duplicate items thats in the attemptlogs to get the real count of the correct answer.
This PR is work on progress. I'm still figuring out why I'm getting 12 correct answer in the exam attempt of `Junaid Shaikh`

I have this comparison score with my changes. 
This is the original.
screen shot 2018-03-23 at 7 54 26 pm
This is with my changes.
screen shot 2018-03-23 at 7 55 14 pm

### References
This fixes #3058 

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by mrpau-richard** [comment]
@rtibbles Now i know the reason why `Junaid Shaikh` exam report has a 80% 12/15 score and on the showExamReportDetailPage it is only 73% 11/15, it's because on the database one item id has two entry with different correct value at the examAttemptLog.
screen shot 2018-03-26 at 10 40 21 pm


This is from the `Junaid Shaikh` exam report
screen shot 2018-03-26 at 11 08 10 pm

This is from the showExamReportDetailPage
screen shot 2018-03-26 at 11 05 33 pm

Should we use the most updated time stamp to get the correct answer?

**Comment by mrpau-richard** [comment]
I use the latest `completion_timestamp` to get the exact examattemptlog.
This is from the Junaid Shaikh exam report
screen shot 2018-03-27 at 11 48 18 pm
And this is his exam report detail.
screen shot 2018-03-27 at 11 56 02 pm

@jonboiser & @christianmemije and @rtibbles  can you guys reviews this PR, Thanks.

File: kolibri/core/assets/src/exams/utils.js [hunk 1]
```diff
@@ -74,18 +74,30 @@ function getExamReport(store, examId, userId, questionNumber = 0, interactionInd
...
-              const attemptLog = examAttempts.find(
+              const attemptLog = examAttempts.filter(
                 log => log.item === question.itemId && log.content_id === question.contentId
-              ) || {
-                interaction_history: [],
-                correct: false,
-                noattempt: true,
-              };
+              );
+              let examAttempLog = attemptLog[0];
```
Comment (jonboiser) [inline kolibri/core/assets/src/exams/utils.js]: typo here `examAttempLog` -> `examAttemptLog`

**Comment by mrpau-richard** [comment]
@jonboiser done fixing the typo error.
By the way to manually test this PR, import the database uploaded at #3058 and import the content of this channel `8fc515793aad4e52b490b03c48ad21de`

**Comment by indirectlylit** [comment]
I haven't manually tested this, but thanks @mrpau-richard

**Pull request merged**